CONTRIBUTING.md

```diff
@@ -45,22 +45,132 @@ To add a new plugin you need to do the following.
+| **extraOptions**                 | Extra options for the plugin.                                                                                                                                                                                | No       | `{}`             |
+| **extraPackages**                | Extra packages to include.                                                                                                                                                                                   | No       | `[]`             |
+| **extraPlugins**                 | Extra plugins to include.                                                                                                                                                                                    | No       | `[]`             |
+| **globalPrefix**                 | Global prefix for the settings.                                                                                                                                                                              | No       | `""`             |
+| **imports**                      | A list of imports for the plugin.                                                                                                                                                                            | No       | `[]`             |
+| **isColorscheme**                | Flag to indicate if the plugin is a colorscheme.                                                                                                                                                             | No       | `false`          |
+| **maintainers**                  | The maintainers of the plugin.                                                                                                                                                                               | No       | `[]`             |
+| **optionsRenamedToSettings**     | List of options renamed to settings.                                                                                                                                                                         | No       | `[]`             |
+| **originalName**                 | The original name of the plugin, usually the plugin's github repo name. E.g. `"foo-bar.nvim"`.                                                                                                               | No       | `name` parameter |
+| **package**                      | The package for the plugin.                                                                                                                                                                                  | No       | `name` parameter |
```

Comment:
Minor inconsitency: the neovem version has a long description with an example. Should this one be the same, seeing as the implementation is the same?

> ``The nixpkgs package attr for this plugin. Can be a string, a list of strings, a module option, or any derivation. For example, "foo-bar-nvim" for `pkgs.vimPlugins.foo-bar-nvim`, or `[ "hello" "world" ]` will be referenced as `pkgs.hello.world`.``

Reply:
Good catch, should definitely be the same.